### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#471] [Esami & Esoneri M-Z]
### Traccia

<img width="935" alt="image" src="https://github.com/user-attachments/assets/ffddf6b5-f787-4b83-8dbb-335477a9f341">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/3e9d2301-3a03-4925-9d2e-e648dba28020)
